server/app.ts

```diff
@@ -140,6 +140,7 @@ export default function createApp(userService: UserService): express.Application
 
   // GovUK Template Configuration
   app.locals.asset_path = '/assets/'
+  app.locals.applicationName = 'HMPPS Template Typescript'
```

Comment:
`HMPPS Typescript Template` sounds better?

Reply:
done that